pkg/drivers/bess/bess.go

```diff
@@ -255,13 +291,34 @@ func (d *Driver) createSwitch(module *network.Switch) error {
 	// Egress gates from original Switch module will be used for L2Forward lookup match, and then
 	// also used for replicator flooding. Add Egress Gate for L2Forward -> replicator and Ingress
 	// gate for reverse
-	l2Fwd.EGates[999] = rep
+	l2Fwd.EGates[0] = rep
```

Comment:
I think this works because the Render is called in agent.go before egress ports are added to the pipeline. That way we know that no egress gates on this switch have been reserved yet. I think it would be good to add a comment here stating this to avoid issues in the future.

Reply:
Ack.